build.gradle

```diff
@@ -60,8 +57,6 @@ subprojects {
                 }
             }
         }
```

Comment:
It feels like this `publishing` block should probably be consolidated into `publishing.gradle`. Will do that, checking that there's not some weird side-effect in the set of generated artifacts.

Reply:
Done